From 097a0271528bd18acb648a0d8262924b06e6dec3 Mon Sep 17 00:00:00 2001
From: Kevin Ottens <ervin@kde.org>
Date: Mon, 20 Jul 2015 16:52:37 +0200
Subject: [PATCH 07/31] Adjust to cmake policy change

Note the values here are likely bogus now, but they were totally bogus
before. I doubt anyone uses them.
---
 gpgme++/CMakeLists.txt | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/gpgme++/CMakeLists.txt b/gpgme++/CMakeLists.txt
index 302e179..1ad3859 100644
--- a/gpgme++/CMakeLists.txt
+++ b/gpgme++/CMakeLists.txt
@@ -58,7 +58,7 @@ if ( GPGME_VANILLA_FOUND )
                                   SOVERSION   ${_gpgmepp_soversion}
                                   OUTPUT_NAME gpgme++
    )
-   get_target_property( GPGMEPP_VANILLA_LIBRARY gpgmepp LOCATION )
+   set(GPGMEPP_VANILLA_LIBRARY ${LIBRARY_OUTPUT_PATH}/libgpgmepp.so)
    install(TARGETS gpgmepp EXPORT kdepimlibsLibraryTargets ${INSTALL_TARGETS_DEFAULT_ARGS} COMPONENT Devel)
 
 endif()
@@ -71,7 +71,7 @@ if ( GPGME_GLIB_FOUND )
                                        SOVERSION   ${_gpgmepp_soversion}
                                        OUTPUT_NAME gpgme++-glib DEFINE_SYMBOL gpgmepp_EXPORTS
    )
-   get_target_property( GPGMEPP_GLIB_LIBRARY gpgmepp LOCATION )
+   set(GPGMEPP_GLIB_LIBRARY ${LIBRARY_OUTPUT_PATH}/libgpgmepp.so)
    install(TARGETS gpgmepp-glib EXPORT kdepimlibsLibraryTargets ${INSTALL_TARGETS_DEFAULT_ARGS} )
 endif()
 
@@ -87,7 +87,7 @@ if ( GPGME_QT_FOUND )
                                      SOVERSION   ${_gpgmepp_soversion}
                                      OUTPUT_NAME gpgme++-qt DEFINE_SYMBOL gpgmepp_EXPORTS
    )
-   get_target_property( GPGMEPP_QT_LIBRARY gpgmepp LOCATION )
+   set(GPGMEPP_QT_LIBRARY ${LIBRARY_OUTPUT_PATH}/libgpgmepp.so)
    install(TARGETS gpgmepp-qt EXPORT kdepimlibsLibraryTargets ${INSTALL_TARGETS_DEFAULT_ARGS} )
 endif()
 
@@ -100,7 +100,7 @@ if ( GPGME_PTHREAD_FOUND )
                                           SOVERSION   ${_gpgmepp_soversion}
                                           OUTPUT_NAME gpgme++-pthread DEFINE_SYMBOL gpgmepp_EXPORTS
    )
-   get_target_property( GPGMEPP_PTHREAD_LIBRARY gpgmepp LOCATION )
+   set(GPGMEPP_PTHREAD_LIBRARY ${LIBRARY_OUTPUT_PATH}/libgpgmepp.so)
    install(TARGETS gpgmepp-pthread EXPORT kdepimlibsLibraryTargets ${INSTALL_TARGETS_DEFAULT_ARGS} )
 endif()
 
@@ -112,7 +112,7 @@ if ( GPGME_PTH_FOUND )
                                       SOVERSION   ${_gpgmepp_soversion}
                                       OUTPUT_NAME gpgme++-pth DEFINE_SYMBOL gpgmepp_EXPORTS
    )
-   get_target_property( GPGMEPP_PTH_LIBRARY gpgmepp LOCATION )
+   set(GPGMEPP_PTH_LIBRARY ${LIBRARY_OUTPUT_PATH}/libgpgmepp.so)
    install(TARGETS gpgmepp-pth EXPORT kdepimlibsLibraryTargets ${INSTALL_TARGETS_DEFAULT_ARGS} )
 endif()
 
-- 
2.5.0

